content/accessibility/accessibility-at-github.mdx

```diff
@@ -0,0 +1,50 @@
+
+When you consider accessibility from the beginning of the design process and incorporate it throughout, it becomes part of an established design process that seeks to include everyone, of all abilities, and allows everyone to achieve more.
+
+**Note:** When reading through accessibility information, you might find the common abbreviation: "a11y". A11y is a shortening of the word "accessibility". It starts with the first letter, "a", uses the number 11 for the eleven characters inbetween the first and last letter, and ends with the last letter, "y".
+
+## What standards you must aim for 
+GitHub aims for [Web Content Accessibility Guidelines (WCAG) 2.1](https://www.w3.org/TR/WCAG21/) AA compliance. This includes all of WCAG 2.0 AA plus additional considerations. 
+
+Currently, [Section 508](https://www.access-board.gov/guidelines-and-standards/communications-and-it/about-the-ict-refresh/overview-of-the-final-rule) (required by law in the United States) follows WCAG 2.0 AA guidelines for compliance. 
+
+## Who should you consider when designing? 
+There are a whole host of disabilities to consider, though the majority will fall into one of these categories: 
+- **Visual:** Anything that deals with sight, or technology that helps someone to see. This includes people who have different types of sight like color blindness.
+- **Cognitive:** Concentration, memory, judgement, problem solving, logic skills. 
+- **Mobility:** Anything that affects movement in a body. 
+- **Hearing:** A spectrum of disabilities related to sound or audio.
+
+Each of these disability categories can be divided into three further sub-categories:
+- **Permanent:** Disability does not go away.
+  - Example: Someone who is deaf
+- **Temporary:** The disability will go away in time.
+  - Example: Someone who has an ear infection
+- **Situational:** The environment someone is in plays a huge factor in how they interact with web content. 
+  - Example: Someone who is at a loud sporting event
+
+For an informative list on real-life disability situations, check [An Alphabet of Accessibility Issues](https://the-pastry-box-project.net/anne-gibson/2014-july-31), published in The Pastry Box Project.
+
+Microsoft has created a downloadable PDF that looks into the area of inclusive design from a people perspective, with several informative examples. Navigate to the ‘Inclusive 101’ toolkit on the main page of [Microsoft’s Inclusive Design website](https://www.microsoft.com/design/inclusive/).
+
+## Internal resources
+- Contact these teams when looking for help:
+  - [@github/accessibility](https://github.com/orgs/github/teams/accessibility) - GitHub staff interested in accessibility related topics and work on website accessibility issues. 
+  - [@github/colorblind](https://github.com/orgs/github/teams/colorblind) - GitHub staff interested in accessibility for colorblindness.
```

Comment:
I'd like to remove these as resources. They were in the original documentation, but there isn't any activity on these teams and I don't know that they're actively monitored? The most responsive place to get feedback is the slack accessibility channel which is listed below.